Cargo.toml

```diff
@@ -54,7 +55,7 @@ features = ["all"]
 libc = "0.2.150"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52"
+version = "0.59"
```

Comment:
This is a breaking change, can't accept that at the moment, see #526

Reply:
Do you need a volunteer to implement https://github.com/rust-lang/socket2/issues/526#issuecomment-2557143907? I'm happy to take a stab at it, or pass the feedback to someone internally at MSFT